src/templates/xpath_validation_errors.html

```diff
@@ -1,10 +1,7 @@
 <div class="alert alert-danger">
     <h4><i class="fa fa-warning"></i> Validation Failed</h4>
-    <p>Please fix all errors before leaving this page:</p>
-    <%=errors%>
-    <ul>
-    <% _.each(errors, function (error) { %>
-        <li><%=error%></li>
-    <% }); %>
```

Comment:
Did this code list the errors twice, or was this iteration for a different kind of error that's not printed by `<%=errors%>`?